src/constants/blacklistConfig.js

```diff
@@ -0,0 +1,93 @@
+const entity = {
+	create: ['id', 'allow_filtering', 'created_at', 'updated_at', 'created_by', 'updated_by'],
+	update: ['id', 'allow_filtering', 'created_at', 'updated_at', 'created_by', 'updated_by'],
+}
+
+const entityType = {
+	create: ['id', 'allow_filtering', 'created_at', 'updated_at', 'created_by', 'updated_by'],
+}
+
+const feedback = {
+	submit: ['id', 'created_at', 'updated_at', 'created_by', 'updated_by'],
+}
+
+const form = {
+	create: ['id', 'organization_id', 'version', 'created_at', 'updated_at', 'created_by', 'updated_by'],
+}
+
+const notification = {
+	create: ['id', 'organization_id', 'created_at', 'updated_at', 'created_by', 'updated_by'],
+	update: ['id', 'organization_id', 'created_at', 'updated_at', 'created_by', 'updated_by'],
+}
+
+const profile = {
+	create: [
+		'user_id',
+		'rating',
+		'visibility',
+		'visible_to_organizations',
+		'external_session_visibility',
```

Comment:
@nevil-mathew password we should not allow  to update, please check in user service also

Reply:
mentoring service is not storing password, it should be rejected in user service